clients/unshaded/src/main/java/tachyon/client/file/FileOutStream.java

```diff
@@ -199,6 +200,7 @@ public void write(int b) throws IOException {
           getNextBlock();
         }
         mCurrentBlockOutStream.write(b);
+        mBytesWritten++;
```

Comment:
should this be incremented at the end of the method in case mShouldCacheCurrentBlock is false?